heed/src/iterator/range.rs

```diff
@@ -199,7 +199,7 @@ impl<KC, DC, C, IM> fmt::Debug for RoRange<'_, KC, DC, C, IM> {
     }
 }
 
-unsafe impl<KC, DC, C, IM> Send for RoRange<'_, WithoutTls, KC, DC, C, IM> {}
+unsafe impl<KC, DC, C, IM> Send for RoRange<'_, KC, DC, C, IM> {}
```

Comment:
Hum... Do we want to keep this impl and the others that are similar?

Reply:
Moving Iterators is unsafe if the transaction uses TLS (`WithTls`). As we can no longer ensure that because we don't have the information in the type anymore, I wonder if we should reintroduce the information (and therefore a correct impl of this trait) or simply remove it (breaking).

Reply:
Ok, [we will have to reintroduce this `Send` impl](https://github.com/meilisearch/heed/issues/213) only when the underlying `RoTxn` is `Send` (`WithoutTls`).

Reply:
I reintroduced the possibility to move iterators between threads when `RoTxn` are using `WithoutTls`.